about.html

```diff
@@ -0,0 +1,29 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>About Me</title>
+</head>
+<body>
+    <nav>
+        <h1><a href="/index.html">Adelle's Smoothies</a></h1>
```

Comment:
links are almost fixed. just add a . before the / and they will all work as expected!

<a href="/index.html"> -> <a href="./index.html">

Reply:
Thanks! Alternatively, would the links work if I just removed the slash?

Reply:
I believe so ya!